.github/workflows/e2e-nightly-1x.yml

```diff
@@ -0,0 +1,86 @@
+# Runs randomly generated E2E testnets nightly on the v1.x branch.
```

Comment:
remove ?

Reply:
The entire file you mean?

Reply:
yeah. it should not be a part of v1 branch.